var/spack/repos/builtin/packages/py-bluepyemodelnexus/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyBluepyemodelnexus(PythonPackage):
+    """Python library to optimize and evaluate electrical models (Nexus addon)."""
+
+    homepage = "https://bbpgitlab.epfl.ch/cells/bluepyemodelnexus"
+    git = "ssh://git@bbpgitlab.epfl.ch/cells/bluepyemodelnexus.git"
```

Comment:
The bbpgitlab URL indicates that this isn't open-sourced - is that something that's in progress? And does py-entity-management join the open-source club as well?

If not, both of these packages will need to go under `bluebrain/repo-bluebrain/packages` instead of the spack builtin repository.

If it is in progress, I propose not to merge this until 1) py-bluepyemodel has been merged and 2) py-bluepyemodelnexus, py-icselector and py-entity-management (with dependencies because this one, too, has internal dependencies) have been opensourced with their code on github.

Reply:
You are correct, both of these packages aren't open-sourced yet, I have just moved them under `bluebrain/repo-bluebrain/packages`